src/components/app.js

```diff
@@ -46,7 +46,7 @@ export default class App extends Component {
             No results found for <strong>{`"${searchQuery}"`}</strong>.
           </p>
         ) : (
-          <ul data-test-search-results-list>
+          <ul data-test-search-results-list>  
```

Comment:
certainly not a blocker, but we've got two trailing spaces here.  we saw a similarly minor typo nail us yesterday in that one test.  gonna look into whether or not ESLint can be configured to scrub this in JSX